### ISSUE TITLE
WP ClI pass-through breaks down with less error on "demyx wp domain.tld help"

### ISSUE THREAD

**Issue by NuclearMonster** [demyxco/demyx#11]
Using a fresh demyx 1.2.3 install from a day or so ago, WP CLI pass-through cannot handle the "help" argument, here is the typical output:
```

user@host:~$ demyx wp domain.tld help
less: unrecognized option: r
BusyBox v1.31.1 () multi-call binary.

Usage: less [-EFIMmNSRh~] [FILE]...

View FILE (or stdin) one screenful at a time

        -E      Quit once the end of a file is reached
        -F      Quit if entire file fits on first screen
        -I      Ignore case in all searches
        -M,-m   Display status line with line numbers
                and percentage through the file
        -N      Prefix line number to each line
        -S      Truncate long lines
        -R      Remove color escape codes in input
        -~      Suppress ~s displayed past EOF


```

**Comment by demyxco** [comment]
Interesting, that's definitely a new error. Thanks for this, I'll have this fixed on the next release.

**Comment by demyxco** [comment]
Fixed in 1.2.4, it appears wp-cli was missing a certain environment variable in an Alpine Linux environment: https://github.com/demyxco/demyx/commit/ee941a3b219a2b686e2735d18b679923d679baac
